flintrock/config.yaml.template

```diff
@@ -5,6 +5,10 @@ services:
     # git-repository:  # optional; defaults to https://github.com/apache/spark
   hdfs:
     version: 2.7.1
+    # optional; defaults to
+    # http://www.apache.org/dyn/closer.lua/hadoop/common/hadoop-{v}/hadoop-{v}.tar.gz?as_json
+    # and must contain a {v} template corresponding to the version
+    # download-source:
```

Comment:
Hmm, I just realized that another requirement we have is that the file be a `.tar.gz`. Could we update the comment to the following?

```
    # optional; defaults to download from a dynamically selected Apache mirror
    # must contain a {v} template corresponding to the version; must be a .tar.gz file
    # download-source: "https://www.example.com/files/hadoop/{v}/hadoop-{v}.tar.gz"
```